### DISCUSSION TITLE
Etsy Listing's Files Download Issue

### DISCUSSION THREAD

**Discussion by ambikesh-cedcommerce** [etsy/open-api#1135] [Q&A]
Hello, Developers,

I hope this message finds you well. I am reaching out to inquire if anyone has encountered challenges while attempting to download files from Etsy, as outlined in the API documentation [here](https://developers.etsy.com/documentation/reference/#operation/getListingFile). According to the documentation, it is possible to retrieve listing files. However, my attempts have only yielded basic information such as:
{
    "listing_file_id": 1,
    "listing_id": 1,
    "rank": 0,
    "filename": "string",
    "filesize": "string",
    "size_bytes": 0,
    "filetype": "string",
    "create_timestamp": 946684800,
    "created_timestamp": 946684800
}

The source for downloading the actual files was not provided. I would greatly appreciate it if any of you could shed some light on this matter.

Thank you for your time and consideration. Looking forward to hearing from you soon.

**Comment by arounde** [comment]
Downloading attached files is not yet a feature. See this thread:
https://github.com/etsy/open-api/discussions/754
